ram/wram.asm

```diff
@@ -1743,7 +1743,7 @@ wOptions:: db
 
 wObtainedBadges:: flag_array NUM_BADGES
 
-	ds 1
+wUnusedObtainedBadges:: db ; written but not read (see init_player_data.asm)
```

Comment:
Put `assert wObtainedBadges + 1 == wUnusedObtainedBadges` above the `ld [hl], a` and remove this comment.

Reply:
Also `flag_array NUM_BADGES` instead of `db` perhaps?